alert-policies/lampy/HighMemoryUsage.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: High Memory Usage (MB)
+
+# Description and details
+description: |+
+  This alert is triggered when the server memory is higher than 90 MB for 5 minutes.
```

Comment:
before: `This alert is triggered when the server memory is higher than 90 MB for 5 minutes.`
can be changed to: `This alert is triggered when the server memory exceeds 90 MB for 5 minutes.`